### PR TITLE
Fix python builds

### PR DESCRIPTION
**Pull request by gchickma**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}

### CODE REVIEW

File: scripts/build/compile-python.sh [hunk 1]
```diff
@@ -22,39 +22,41 @@ export PIP_CONFIG_FILE=$PIP_CONF
...
+	# 	    pip install -r requirements.txt
+	# 		RESULT=$?
+	# 		if [ $RESULT -ne 0 ] ; then
+	# 			exit 89
+	# 		fi
+	# 	else
+	# 	    echo "No requirements.txt file found in project"
+	# 	fi
+	# fi
+	echo "Python 2 no longer supported ..."
```
Comment (timrbula) [inline scripts/build/compile-python.sh:58]:
```suggestion
	echo "Python 2 no longer supported..."
```

File: scripts/common/initialize-dependencies-python.sh [hunk 2]
```diff
@@ -10,14 +10,23 @@ if [ "$DEBUG" == "true" ]; then
 fi
 
 if [ "$BUILD_LANGUAGE_VERSION" == "2" ]; then
-    echo "Installing Python 2 ..."
-    apk add python python-dev py-pip
+    # echo "Installing Python 2 ..."
+    # apk add python python-dev py-pip
+    #
+    # pip install --upgrade pip
+    echo "Python 2 no longer supported ..."
```
Comment (timrbula) [inline scripts/common/initialize-dependencies-python.sh:17]:
```suggestion
    echo "Python 2 no longer supported..."
```

File: scripts/common/initialize-dependencies-python.sh [hunk 3]
```diff
@@ -10,14 +10,23 @@ if [ "$DEBUG" == "true" ]; then
...
+    exit 89
 elif [ "$BUILD_LANGUAGE_VERSION" == "3" ]; then
     echo "Installing Python 3 ..."
```
Comment (timrbula) [inline scripts/common/initialize-dependencies-python.sh:20]:
```suggestion
    echo "Installing Python 3..."
```

File: scripts/common/initialize-dependencies-python.sh [hunk 4]
```diff
@@ -10,14 +10,23 @@ if [ "$DEBUG" == "true" ]; then
...
-    apk add python3 python3-dev
+    apk add python3 python3-dev py3-pip
+
+    pip3 install --upgrade pip
 else
     echo "Defaulting to and installing Python 3 ..."
```
Comment (timrbula) [inline scripts/common/initialize-dependencies-python.sh:25]:
```suggestion
    echo "Defaulting to and installing Python 3..."
```

File: scripts/common/initialize-dependencies-python.sh [hunk 5]
```diff
@@ -10,14 +10,23 @@ if [ "$DEBUG" == "true" ]; then
...
-    apk add python3 python3-dev
+    apk add python3 python3-dev py3-pip
+
+    pip3 install --upgrade pip
 fi
 
-apk add gcc g++
\ No newline at end of file
+echo "Installing additional tools & libraries ..."
```
Comment (timrbula) [inline scripts/common/initialize-dependencies-python.sh:31]:
```suggestion
echo "Installing additional tools & libraries..."
```

**Review by timrbula — CHANGES_REQUESTED** [review]
Some extreme nitpicking w/ suggestions to remove the space before the ellipsis (I think that is how we do it in other places?)

The other nit is that do we need to have the old code commented out or can it just be removed?

**Comment by gchickma** [comment]
@timrbula Both fair comments (I intended to remove the commented out code but forgot)

Updated branch with recommended changes

**Review by timrbula — APPROVED** [review]

**Pull request merged**